x/delegation/keeper/delegation_state.go

```diff
@@ -177,9 +183,9 @@ func (k *Keeper) GetSingleDelegationInfo(ctx sdk.Context, stakerID, assetID, ope
 func (k *Keeper) GetDelegationInfo(ctx sdk.Context, stakerID, assetID string) (*delegationtype.QueryDelegationInfoResponse, error) {
 	var ret delegationtype.QueryDelegationInfoResponse
 	ret.DelegationInfos = make(map[string]*delegationtype.DelegationAmounts, 0)
```

Comment:
equals to `make(string]*delegationtype.DelegationAmounts)`

Reply:
right, I will change it in the next commit.